### DISCUSSION TITLE
Using a Library Chromebook

### DISCUSSION THREAD

**Discussion by The-star-ship** [Cyfrin/foundry-full-course-cu#2056] [Q&A]
At the moment, I only have access to a library Chromebook and Hotspot.  Not sure about the safety or technical limitations of trying to set up a wallet on a public device.  The browser automatically resets every time I log out.  Is there a work around or am I just limited on what I can do?  Is it even possible to set up a wallet?

**Comment by EngrPips** [comment]
Hey @The-star-ship, You can set up a wallet that you will only use for learning as you will never put real funds there, so if it gets comprehended, it should not be the end of the world. You can set up a real wallet for personal use when you have a more secure personal device.

**Comment by The-star-ship** [comment]
I was hoping that was the case, but the library computer has some restriction that is not allowing me to add the MetaMask extension to the chrome browser. I was fiddling with it to find a work around. If you have any suggestions, that would be great!

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by EngrPips** [reply]
Honestly, I have no experience with Chromebook, and I don't even know what that is. I am sorry I can't help here, and I hope you get a solution soon.
